Update query packs to not use deprecated fields

tas50 opened the pull request:
- RDS clusters and instances were renamed
- Remove a duplicate ElastiCache query that used the old field

HRouhani commented:
@tas50 
any reason you removed:
```
  - uid: mondoo-asset-count-aws-elasticache-clusters
    title: AWS ElastiCache Clusters
    mql: aws.elasticache.clusters.length
```

This is the reason the pipeline failed!

tas50 commented:
@HRouhani I updated the policy to remove that reference. We had 2 queries that were actually the same query. One used the original resource name and one used the new name. I removed the deprecated one.

HRouhani reviewed (approved):
Thanks Tim
